crates/red_knot_python_semantic/src/types/call.rs

```diff
@@ -4,15 +4,18 @@ use crate::Db;
 
 mod arguments;
 mod bind;
-pub(super) use arguments::{Argument, CallArguments};
+pub(super) use arguments::{Argument, ArgumentForm, ArgumentKind, CallArguments};
 pub(super) use bind::Bindings;
 
 /// Wraps a [`Bindings`] for an unsuccessful call with information about why the call was
 /// unsuccessful.
 ///
 /// The bindings are boxed so that we do not pass around large `Err` variants on the stack.
-#[derive(Debug, Clone, PartialEq, Eq)]
-pub(crate) struct CallError<'db>(pub(crate) CallErrorKind, pub(crate) Box<Bindings<'db>>);
+#[derive(Debug, Clone)]
+pub(crate) struct CallError<'call, 'db>(
```

Comment:
Do we need the precision of both lifetimes or could we use one unified `'a` lifetime for both `call` and `db`?

It seems to me that `call` will always be the shorter lifetime of the two.

Reply:
The extra lifetime went away with the new representation